zerver/lib/markdown/help_relative_links.py

```diff
@@ -66,9 +66,32 @@ def stream_handle_match(key: str) -> str:
     return stream_instructions_no_link
 
 
+draft_instructions = """
+1. Click on <i class="fa fa-pencil"></i> **Drafts** in the left sidebar.
+"""
+
+scheduled_instructions = """
+1. Click on <i class="fa fa-calendar"></i> **Scheduled messages** in the left
+   sidebar. If you do not see this link, you have no scheduled messages.
+"""
+
+message_info = {
+    "drafts": ["Drafts", "/#drafts", draft_instructions],
+    "scheduled": ["Scheduled messages", "/#scheduled", scheduled_instructions],
+}
+
+
+def message_handle_match(link_type: str) -> str:
+    if relative_help_links:
+        return f"1. Go to [{message_info[link_type][0]}]({message_info[link_type][1]})."
+    else:
+        return message_info[link_type][2]
```

Comment:
To be consistent with the other "handle_match" functions, maybe "key" instead of "link_type" for the parameter here?